docs/repositories.md

```diff
@@ -0,0 +1,71 @@
+# Generated Bazel repositories
+
+rules_nodejs produces several repositories for you to reference.
+Bazel represents your Workspace as one repository, and code fetched or installed from outside your workspace lives in other repositories.
+These are referenced with the `@repo` syntax in your BUILD files.
+
+## @nodejs
+
+This repository is created by calling the `node_repositories` function. It contains the node, npm, and yarn programs.
```

Comment:
nit: "function in your WORKSPACE file". First time users will not know where to look.